internal/strategies/helpers.go

```diff
@@ -12,6 +13,20 @@ import (
 
 var VariantNormalizationSeed uint32 = 86028157
 
+var colonByte = []byte(":")
+
+var hashPoolSeed0 = sync.Pool{
```

Comment:
I believe this is used in rollout strategies, can we call it that rather than `hashPoolSeed0`? I had to go and look why we need a 0 seed